tests/rptest/services/storage.py

```diff
@@ -154,11 +154,33 @@ class PartitionNotFoundError(Exception):
     pass
 
 
+class NodeCacheStorage:
```

Comment:
This can be annotated with `@dataclasses.dataclass`, to avoid defining `__init__` and `__str__` etc

Reply:
done